x/ibc/testing/chain.go

```diff
@@ -200,6 +200,7 @@ func (chain *TestChain) NextBlock() {
 
 // SendMsg delivers a transaction through the application. It updates the senders sequence
 // number and updates the TestChain's headers.
+// TODO: update SignCheckDeliver to use tx generator
```

Comment:
@fedekunze what would be the best way to do this for ibc

Reply:
cc: @colin-axner

Reply:
I'm not familiar with what these changes imply, but I don't think there should be any special treatment for IBC tx's. 

The IBC module should just be treated like any other module. Maybe there might be an issue atm since the 02-client hasn't been migrated to proto?

Other than that you should just be able to pass in the tx generator right?

Reply:
```suggestion
```